templates/stripes/stripe-infographic.php

```diff
@@ -32,8 +43,14 @@
     </div>
     <div class="vssl-stripe--infographic--lightbox">
         <div class="vssl-stripe--infographic--image">
-            <img src="<?= $this->image($image, !empty($image_style) ? $image_style : null) ?>"
-                alt="<?= $image ?>" loading="lazy" />
+            <img
+              src="<?= $this->image(
+                $image,
+                !empty($image_style) ? $image_style : null
+              ) ?>"
+              alt="<?= $image_alt ?? ($image ?? "") ?>"
```

Comment:
Using new `$image_alt` property for the alt attribute, defaulting to the image path if not set